proto/ibc/core/channel/v1/tx.proto

```diff
@@ -47,11 +47,11 @@ enum ResponseResultType {
   option (gogoproto.goproto_enum_prefix) = false;
 
   // Default zero value enumeration
-  RESPONSE_RESULT_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "UNSPECIFIED"];
+  RESPONSE_RESULT_TYPE_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "UNSPECIFIED"];
```

Comment:
I copied over the changes that are on main. There is some linting issues for channel protos on the `02-client-refactor` branch that are fixed on main. In order to add the `deprecated` flag for `MsgSubmitMisbehavior` I fixed this (the linter was failing otherwise).

Reply:
Nice 👍 I fixed this on main in #1143 before but I think those changes have been pulled into `02-client-refactor` yet. 
I think we should probably merge the latest of main in, also includes the fixes for gh actions (link checker)